cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -162,6 +162,20 @@ Example tags:
 
 - `k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage`: `100G`
 
+ASG labels can specify autoscaling options, overriding the global cluster-autoscaler
+settings for the labeled ASGs. Those labels takes the same values format as the
+cluter-autoscaler command line flags they override (a float or a duration, encoded
```

Comment:
```suggestion
cluster-autoscaler command line flags they override (a float or a duration, encoded
```

Reply:
Thanks for catching this, updated & rebased accordingly!